cf/commands/route/create_route.go

```diff
@@ -29,13 +31,15 @@ func init() {
 }
 
 func (cmd *CreateRoute) MetaData() command_registry.CommandMetadata {
+	contextPathHelp := T(`Optional context path. Path must include at least one character following a leading forward slash (/). Trailing slashes will be stripped. Requests received with a trailing slash will match.`)
```

Comment:
Having discussed the meaning of the message with Shannon and Nic, we'd like to suggest a small change to:

`Optional context path. Path must include at least one character following a leading forward slash (/). Trailing slashes will be stripped, but requests received with a trailing slash will match.`

We'll pull it in after your update.